index.js

```diff
@@ -1,6 +1,86 @@
 function loader (mcVersion) {
+  const Item = require('prismarine-item')(mcVersion)
+  const Window = require('./lib/Window')(Item)
+
+  let windows
+  if (mcVersion.startsWith('1.14')) {
```

Comment:
would be better to retrieve majorVersion from mcData, this check doesn't work for snapshot

Reply:
The real fix would be to put the json in mcData and get the one for the right version.